include/EVT/io/pin.hpp

```diff
@@ -128,6 +128,10 @@ enum class Pin {
 #endif
 
 #ifdef STM32F4xx
+    SPI_SCK = PA_9,// D8
```

Comment:
Why did you label two of them with their D values but not the other two?

Reply:
well... i meant to talk to you about that but clearly forgot. The other 2 don't have a D label. Only SPI1 has the capabilities to be only on D's, but SPI1's necessary pins are being used by GPIO by default, so it isn't usable normally. I'll talk to you about it at EVT tonight.

Reply:
SPI1 is now the default SPI being used. Going to test it on thurs but everything should be in order.